presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -1426,8 +1413,7 @@ public void testMaterializedViewOptimizationWithDerivedFieldsWithAlias()
         }
     }
 
-    //FIXME: It does not work as column map in the materialized View metadata is not correct. It only contains 1 map instead of 2.
-    // https://github.com/prestodb/presto/pull/15996
+    //TODO: Populate columnMappings to cover all joined base tables, https://github.com/prestodb/presto/issues/16220
     @Test(enabled = false)
```

Comment:
Can we enable this?

Reply:
Fixing testMaterializedViewForJoin needs more complete population of the columnMappings structure. The change is orthogonal to REFRESH and deserves a standalone PR. Filing the issue #16220 and updating the TODO.

Reply:
Leave the space before TODO